pylzy/lzy/api/v1/remote/lzy_service_client.py

```diff
@@ -131,8 +127,14 @@ async def __start(self):
         self.__ops_stub = LongRunningServiceStub(CHANNEL)
 
     @redefine_errors
-    @retry(config=RETRY_CONFIG, action_name="starting workflow")
-    async def start_workflow(self, name: str, storage: Storage, storage_name: str) -> str:
+    @retry(config=RETRY_CONFIG, action_name="start workflow")
+    async def start_workflow(
+            self,
```

Comment:
It is unimportant, but everywhere in such places, we have 4-space indent instead of 8-spaces.

Reply:
do we have editor config or smth like that to autovalidate this rule?

Reply:
Well
I have this setting on my [vimrc](https://github.com/vhaldemar/dotfiles/blob/master/vimrc#L121)
Probably there is something for pycharm too, like https://www.jetbrains.com/help/pycharm/code-style-python.html#tabs-and-indents-tab

Reply:
I mentioned it because in java's `Lzy` we have checkstyle.xml which can be imported into IDE and all settings are autowired. Well for python client I do this manually

Reply:
Maybe you can do it also for the Pycharm, but I'm using vim.
I can share my config if you want :D